src/OrchardCore.Modules/OrchardCore.Localization/Startup.cs

```diff
@@ -35,6 +43,8 @@ public override void ConfigureServices(IServiceCollection services)
                 AddDataAnnotationsPortableObjectLocalization();
 
             services.Replace(ServiceDescriptor.Singleton<ILocalizationFileLocationProvider, ModularPoFileLocationProvider>());
+
+            services.Configure<CultureOptions>(_shellConfiguration.GetSection("OrchardCore_Localization"));
```

Comment:
This way the `CultureOptions` will be there even the localization module is disable

Reply:
And that's why it was fine when I use it in `ILocalizationService` before

Reply:
No because the above startup only run if the `OC.Localization` module is enabled and this is the one that also registers `LocalizationService`, in `AddDefaultServices()` we only register `DefaultLocalizationService`.

Not sure what you wanted to do in `ILocalizationService` but would need to be in both `LocalizationService` and `DefaultLocalizationService`. And if you need to inject `IOptions<CultureOptions>` in both of them, then same problem, here `services.Configure<CultureOptions>()` is only registered if `OC.Localization` is enabled.

But maybe I didn't understand what you meant.